client/my-sites/plans-grid/components/plan-type-selector/types.ts

```diff
@@ -11,6 +11,7 @@ export type PlanTypeSelectorProps = {
 	withDiscount?: string;
 	enableStickyBehavior?: boolean;
 	stickyPlanTypeSelectorOffset?: number;
+	onPlanTypeSelectorChange?: ( selectedItem: { key: SupportedUrlFriendlyTermType } ) => void;
```

Comment:
precisely 🙌 
now we have a proper interface that's reusable for any context, without the complexity of "the component knowing too much"

NIT: wonder if a better name might work for this, something more granular or general to reflect the actual operation? `onSomethingChange` - but nothing comes to mind though, so.. 🤷

Reply:
> wonder if a better name might work for this, something more granular or general to reflect the actual operation?

This makes sense to me since the plan type selector will be part of the plans grid shared package. Perhaps `onPlanTermChange` or `onPlanIntervalChange`... 🤔

Reply:
I decided on onPlanIntervalChange. If we don't like it, I'm happy to switch back or to something else entirely -- it's an easy adjustment 🙂